README.md

```diff
@@ -111,6 +111,41 @@ module.exports = (argv) => {
 };
 ```
 
+## Http proxy
+
+#### Enabling proxy
+
+The server contains a simple http proxy. 
+The proxy must be configured in the config file.
+To enable this proxy:
+
+```sh
```

Comment:
I don't think we need the flag - this feature can only be implemented using a config file and either the user has added `proxy` options or they haven't

Reply:
removed the flag, added check for the presence of proxy config before calling the proxy